tests/ctst/world/Zenko.ts

```diff
@@ -197,13 +257,14 @@ export default class Zenko extends World {
      * @returns {undefined}
      */
     async prepareARWWI(ARWWIName: string, ARWWIPassword: string, ARWWITargetRole: string) {
+
         if (!(ARWWIName in CacheHelper.ARWWI)) {
-            const token = await this.getWebIdentityToken(
+            const token: string = await this.getWebIdentityToken(
```

Comment:
We can specify this type in the function instead
```suggestion
            const token = await this.getWebIdentityToken(
```